packages/app-builder/src/components/Scenario/AstBuilder/AstBuilderNode/Operand/OperandInfos.tsx

```diff
@@ -9,99 +9,90 @@ import {
   getDataTypeTKey,
   getOperatorTypeIcon,
   getOperatorTypeTKey,
-} from './OperandEditor/OperandOption/Option';
+} from './utils';
 
 const MAX_ENUM_VALUES = 50;
 
-interface OperandTooltipProps {
+interface OperandInfosProps {
   children: React.ReactNode;
-  content: React.ReactNode;
-  side?: Tooltip.TooltipContentProps['side'];
-  align?: Tooltip.TooltipContentProps['align'];
-  sideOffset?: Tooltip.TooltipContentProps['sideOffset'];
-  alignOffset?: Tooltip.TooltipContentProps['alignOffset'];
+  className?: string;
+  gutter?: number;
+  shift?: number;
 }
 
-export function OperandTooltip({
+export function OperandInfos({
   children,
-  content,
-  side = 'right',
-  align = 'start',
-  sideOffset,
-  alignOffset,
-}: OperandTooltipProps) {
+  className,
+  gutter,
+  shift,
+}: OperandInfosProps) {
   return (
-    <Tooltip.Root delayDuration={0}>
-      <Tooltip.Trigger tabIndex={-1} asChild>
-        <span>{children}</span>
-      </Tooltip.Trigger>
-      <Tooltip.Portal>
-        <Tooltip.Content
-          side={side}
-          align={align}
-          sideOffset={sideOffset}
-          alignOffset={alignOffset}
-          className="bg-grey-00 border-grey-10 flex max-h-[400px] max-w-[300px] overflow-y-auto overflow-x-hidden rounded border shadow-md"
-        >
-          <div className="flex flex-col gap-2 p-4">{content}</div>
-        </Tooltip.Content>
-      </Tooltip.Portal>
-    </Tooltip.Root>
+    <Ariakit.HovercardProvider
+      showTimeout={0}
+      hideTimeout={0}
+      placement="right-start"
+    >
+      <Ariakit.HovercardAnchor tabIndex={-1}>
+        <Icon icon="tip" className={className} />
+      </Ariakit.HovercardAnchor>
+      <Ariakit.Hovercard
+        gutter={gutter}
+        shift={shift}
+        portal
+        className="bg-grey-00 border-grey-10 flex max-h-[400px] overflow-y-auto overflow-x-hidden rounded border shadow-md"
+      >
+        <div className="flex flex-col gap-2 p-4">{children}</div>
+      </Ariakit.Hovercard>
+    </Ariakit.HovercardProvider>
   );
 }
 
-export const OperandDescription = ({
-  operand,
-}: {
-  operand: {
-    name: string;
-    operandType: LabelledAst['operandType'];
-    dataType: LabelledAst['dataType'];
-    description?: string;
-    values?: EnumValue[];
-  };
-}) => {
+//TODO: change for a polymorphic option with a conditional render on the type
+export const OperandDescription = ({ option }: { option: LabelledAst }) => {
```

Comment:
Prepare to a future model change: directly inject the current node info so the rendering logic can be conditionnal based on the current node type.

In fine: no more optional values, description, aggregation data... for all nodes, just a polumorphic data that can render conditionnally (easier to integrate new UI)